test/e2e/chaos/basic.go

```diff
@@ -49,6 +49,32 @@ import (
 	"github.com/chaos-mesh/chaos-mesh/test/pkg/fixture"
 )
 
+type exceptMap struct {
+	// Source is the source pod labelSelector of chaos
+	Source map[string]string
+	// Target is the target pod labelSelector of chaos
+	Target map[string]string
+}
+
+type exceptMaps []*exceptMap
+
+func (e exceptMaps) isExcept(source, target map[string]string) bool {
```

Comment:
Is there any way to avoid `exceptMaps`? Like combinating `allSlowConnection()` and `allBlockedConnection()` as one function.

It seems `exceptMaps` wants to avoiding throw error in `allSlowConnection()`.

Reply:
Mixing `allSlowConnection` and `allBlockedConnection` into one function sounds not a good idea, so what's this combined function meaning? It's ambiguous!

Adding `excepMaps` would skip some cases while calling `allSlowConnection`, it's not just ONLY suitable to this PR, but also for the other potential issues, and I think this design would be more commonly to use.

Reply:
How about making a method that return a thing like `map[EnumOfNetworkStates][][]int`?

I just do NOT want to skip making asserting on something. 
And more, I think we could improve these functions which could helper us to detect network status.

We pretend that there are three states for network: good, slow and blocked, then
we using these methods during assertion in e2e test. 

Currently `allSlowConnection` is not enough expressively with both network partition and delay. It assumes that network communications between each peer are all good, except few connections is slow. Using `allSlowConnection` with a `expectMaps` just skipping connections which setted with network-partition, then using `allBlockedConnection` to assert the network-partition exists is acceptable. We do not need do any changes if we don't have another brilliant ideas.

Reply:
Got it, I'll try :)